docs/daytona_create.md

```diff
@@ -3,7 +3,7 @@
 Create a workspace
 
 ```
-daytona create [REPOSITORY_URL] [flags]
+daytona create [TOKEN_IDENTITY] [REPOSITORY_URL] [flags]
```

Comment:
`[REPOSITORY_URL]` should be the first param always.

`[TOKEN_IDENTITY]` should be a flag that, if not provided, we assume one on the backend without bothering the user (if possible).